vector/src/main/java/im/vector/app/features/permalink/PermalinkHandler.kt

```diff
@@ -220,6 +220,9 @@ class PermalinkHandler @Inject constructor(private val activeSessionHolder: Acti
         const val MATRIX_TO_CUSTOM_SCHEME_URL_BASE = "element://"
         const val ROOM_LINK_PREFIX = "${MATRIX_TO_CUSTOM_SCHEME_URL_BASE}room/"
         const val USER_LINK_PREFIX = "${MATRIX_TO_CUSTOM_SCHEME_URL_BASE}user/"
+        const val SCHILDI_CHAT_CUSTOM_SCHEME_URL_BASE = "schildichat://"
```

Comment:
Let's use the upstream names for these variables with prepended "SC_":
- SC_MATRIX_TO_CUSTOM_SCHEME_URL_BASE
- SC_ROOM_LINK_PREFIX
- SC_USER_LINK_PREFIX